packages/vscode-extension/src/handlers.ts

```diff
@@ -874,14 +877,37 @@ function checkCoreNotEmpty(): Result<null, SystemError> {
 }
 
 export async function validateAzureDependenciesHandler(): Promise<string | undefined> {
+  // skip debugging if there is already a debug session
+  if (allRunningDebugSessions.size > 0) {
```

Comment:
Consider make this a utility to avoid duplicated code

Reply:
Good advice, updated.